From df69543720de5bf7dd1a734c3bef6d191ceeab33 Mon Sep 17 00:00:00 2001
From: Richard W.M. Jones <rjones@redhat.com>
Date: Wed, 22 Sep 2010 17:43:34 +0100
Subject: [PATCH] regressions: Test just-built guestfish.

Instead of testing the installed /usr/bin/guestfish.

This fixes commit ddda0f7bd00a37274dae38f4ce93955b8cfdf7d7.
---
 regressions/test-guestfish-a.sh |    2 +-
 regressions/test-guestfish-d.sh |    3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/regressions/test-guestfish-a.sh b/regressions/test-guestfish-a.sh
index 8e990ed..8847b85 100755
--- a/regressions/test-guestfish-a.sh
+++ b/regressions/test-guestfish-a.sh
@@ -23,6 +23,6 @@ set -e
 rm -f test.img
 
 truncate -s 10M test.img
-guestfish -a test.img </dev/null
+../fish/guestfish -a test.img </dev/null
 
 rm -f test.img
diff --git a/regressions/test-guestfish-d.sh b/regressions/test-guestfish-d.sh
index b97ad3d..bf5d514 100755
--- a/regressions/test-guestfish-d.sh
+++ b/regressions/test-guestfish-d.sh
@@ -46,7 +46,8 @@ cat > test.xml <<EOF
 </node>
 EOF
 
-guestfish -c "test://$cwd/test.xml" --ro -d guest -x </dev/null >test.out 2>&1
+../fish/guestfish -c "test://$cwd/test.xml" --ro -d guest -x \
+  </dev/null >test.out 2>&1
 grep -sq '^add_drive_ro.*test.img' test.out
 
 rm -f test.img test.xml test.out
-- 
1.7.2.3

